0021-plugin-rpc-api/0021-plugin-rpc-api.md

```diff
@@ -43,181 +66,3 @@ Content-Type: application/json
 
 true
 ```
-
-### Send Request (ILQP)
```

Comment:
I hear that.

I'm still missing why you want to work on an HTTP / JSON version of the LPI instead of LLL.

The decisions to be made are:
- JSON vs OER
- HTTP vs WebSockets + TLS sockets
- Exact same data as current LPI or LPI with some minor changes/improvements

We should pick the set of decisions that we think make the most sense as the base layer for most ILP interactions.

Right now I would argue for:
- OER because you already need an OER parser for the rest of ILP, so it's more consistent with the rest of the stack (and so you don't need to base64-encode the ILP packet), and we picked OER for its parsing speed and relative simplicity as far as binary formats go
- Sockets because HTTP doesn't provide much benefit for a binary protocol 
- Add improvements or changes where we can over the current LPI if we're making changes anyway

That is basically what LLL is. Why is the JSON + HTTP version better? Is it just to be more similar to what we already have implemented?